### ISSUE TITLE
No attribute 'WindowsAzureMissingResourceError' 

### ISSUE THREAD

**Issue by yarnball** [jschneier/django-storages#102]
Hi,

I've not used django-storages with any provider. I keep getting the error 'module' object has no attribute 'WindowsAzureMissingResourceError'.

Here is the traceback http://dpaste.com/12F5D3S

Here is a snippet of my 'settings.py':

```
DEFAULT_FILE_STORAGE = 'storages.backends.azure_storage.AzureStorage'

AZURE_ACCOUNT_NAME = "account_name"

AZURE_ACCOUNT_KEY = "xxxxx"

AZURE_CONTAINER = "container_name"

MEDIA_ROOT = '/media/'

MEDIA_URL = 'stashdimages.core.windows.net/'
```

Models.py

`testfile = models.FileField("Test File", blank=True, null= True)`

**Comment by jschneier** [comment]
Hi and thanks for the report. This sounds like a version issue. Can you tell me what version of azure you have installed?

**Comment by zuck** [comment]
I think the problem is related with not installed dependencies.

Is **azure** Python module installed in your environment?

It seems **django-storages** doesn't automatically install it (as the **dropbox** module for **DropBoxStorage** class).

**Comment by aleccool213** [comment]
If the `azure` pip package needs to be installed alongside why isnt this in the docs?

**Comment by ZuluPro** [comment]
@aleccool213 As django-storage is free software, we'll be happy to receive a contribution from you about it.

**Comment by aleccool213** [comment]
@ZuluPro Thought there might have been a different reason for this :) lol will do 👍

**Comment by sww314** [comment]
Fixed in #565
